### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by darxriggs**
Each commit resolves a dedicated aspect.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -72,19 +74,16 @@
...
-            <url>http://repo.jenkins-ci.org/public/</url>
-        </pluginRepository>
-    </pluginRepositories>
-</project>  
-  
-
+  <repositories>
+    <repository>
+      <id>repo.jenkins-ci.org</id>
+      <url>http://repo.jenkins-ci.org/public/</url>
```
Comment (oleg-nenashev) [inline pom.xml]: Https if you modify it anyway

Comment (darxriggs) [reply pom.xml]: done

File: src/main/resources/hudson/plugins/build_timeout/BuildTimeoutWrapper/help-strategy.groovy [hunk 2]
```diff
@@ -22,43 +22,43 @@
  * THE SOFTWARE.
  */
 
-package hudson.plugins.build_timeout;
+package hudson.plugins.build_timeout
```
Comment (ikedam) [inline src/main/resources/hudson/plugins/build_timeout/BuildTimeoutWrapper/help-strategy.groovy]: +1: Semicolons are optional in Groovy and recommend to be removed.
http://groovy-lang.org/style-guide.html#_no_semicolons

**Comment by ikedam** [comment]
Somehow the build failed on Jenkins, it looks caused not for this change but for some trouble in Jenkins.

**Comment by ikedam** [comment]
The build for the merge commit succeeded.
Everything is OK.
Thanks!

**Comment by ikedam** [comment]
Released in build-timeout-1.19.1.

**Pull request merged**